src/shell/autoexec.cpp

```diff
@@ -509,9 +519,21 @@ void AutoExecModule::AutoMountDrive(const std::string& dir_letter)
 	}
 }
 
-void AutoExecModule::AddLine(const Placement placement, const std::string& line)
+void AutoExecModule::AutoMountDriveC(const std::string& directory)
 {
-	autoexec_lines[placement].push_back(line);
+	constexpr auto placement = Placement::InitialAutogeneratedCommands;
+	AddLine(placement, CmdMount + std::string("-u C") + ToNul);
+	AddLine(placement, CmdMount + std::string("C ") + directory + ToNul);
+	AddLine(placement, "@C:");
+}
+
+void AutoExecModule::AutoMountDriveD(const std::string& cdrom_images)
+{
+	constexpr auto placement = Placement::InitialAutogeneratedCommands;
+	AddLine(placement, CmdMount + std::string("-u D") + ToNul);
+	AddLine(placement,
+	        CmdImgMount + std::string("D ") + cdrom_images +
+	                std::string(" -t iso") + ToNul);
```

Comment:
The `std::string`'s can add literals, so this can be maybe be simplified:

```c++
CmdImgMount + "D " + cdrom_images + " -t iso" + ToNul
```

Reply:
OK, changed.